docs/src/man/getting_started.md

```diff
@@ -23,6 +23,31 @@ Nullable()
 Nullable(1) + Nullable()
 ```
 
+The get() function is very usefull to extract the value from the Nullable wrapper.
```

Comment:
I would just say "can be used to". `Nullable` should be in backticks. Also mention "when it is not null".